### PR TITLE
resolve response-code is set as INTERNAL_SERVER_ERROR in catch block …

### LINKED ISSUE #346: response-code is set as INTERNAL_SERVER_ERROR in catch block for all services
In Controller/[filename].js , there is a catch block that catches if there is any exception thrown in any internal flow of the services.
In all services, the catch block is commonly made the response-code as "INTERNAL_SERVER_ERROR" which would give 500 response code irrespective of the error. The error may be because of wrong input or something not supported or internal-server-error.

})
.catch(async function (responseBody) {
responseBodyToDocument = responseBody;
responseCode = responseCodeEnum.code.INTERNAL_SERVER_ERROR;
let responseHeader = await restResponseHeader.createResponseHeader(xCorrelator, startTime, req.url);
restResponseBuilder.buildResponse(res, responseCode, responseBody, responseHeader);
});

proposal:
We need to analyze possible response codes for all services (both in AP and in all individual applications) and update the response block for error condition.

### PR DESCRIPTION
**Pull request by ManasaBM1**
…for all services

Fixes #346

### CODE REVIEW

File: server/service/individualServices/ForwardingService.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const {
 } = require('onf-core-model-ap/applicationPattern/services/ElasticsearchService');
 const onfAttributes = require('onf-core-model-ap/applicationPattern/onfModel/constants/OnfAttributes');
 const ElasticsearchPreparation = require('./ElasticsearchPreparation');
+const BadRequestHttpException = require('onf-core-model-ap/applicationPattern/rest/server/HttpException');
```
Comment (MartinSunal) [inline server/service/individualServices/ForwardingService.js:7]: We will not use BadRequestHttpException anymore but http-errors library.
Please, wait until required changes are done in controllers and then you can change this service to use errors from http-errors.

**Review by MartinSunal — CHANGES_REQUESTED** [review]

File: server/service/individualServices/ForwardingService.js (hunk 1 above)
Comment (ManasaBM1) [reply server/service/individualServices/ForwardingService.js]: Ok Done

File: server/service/individualServices/ForwardingService.js [hunk 2]
```diff
@@ -33,6 +33,9 @@ class ForwardingService {
         if (response.body.updated === 1) {
             return { "took": response.body.took };
         } else {
+            if(response.body.total === 0){
+               throw new createHttpError.BadRequest("requested uuid does not exit")
```
Comment (MartinSunal) [inline server/service/individualServices/ForwardingService.js:37]: pls, use uuid value as part of message and be more verbose e.g. (CC with uuid ${controlConstructUuid} does not exist.). Also, make sure you format code you changed properly (missing spaces....)
Obviously, this comment applies to other place too.

**Review by MartinSunal — CHANGES_REQUESTED** [review]

**Comment by MartinSunal** [comment]
@ManasaBM1 Are there any other places in this app where we need to return http error? I am asking because this PR contains `Fixes #346` in description which will cause closing of the issue when this PR is merged.

**Comment by ManasaBM1** [comment]
> @ManasaBM1 Are there any other places in this app where we need to return http error? I am asking because this PR contains `Fixes #346` in description which will cause closing of the issue when this PR is merged.

 @MartinSunal Nope I think These are plcaes we need  include http error related basic and individual Services

File: server/service/individualServices/ForwardingService.js (hunk 2 above)
Comment (ManasaBM1) [reply server/service/individualServices/ForwardingService.js]: done

**Review by MartinSunal — APPROVED** [review]

**Pull request merged**